File: src/main/java/com/back/nbe9112team06/global/springDoc/example/AuthApiExamples.java
package com.back.nbe9112team06.global.springDoc.example;

public class AuthApiExamples {

    // ── POST /api/auth/login ──────────────────────────────

    public static final String LOGIN_SUCCESS_JSON = """
            {
              "code": "201-1",
              "message": "로그인 성공",
              "data": {
                "nickname": "gildong"
              }
            }
            """;

    // ── GET /api/auth/me ──────────────────────────────

    public static final String GET_MY_INFO_SUCCESS_JSON = """
            {
              "code": "200-1",
              "message": "조회 성공",
              "data": {
                "nickname": "gildong"
              }
            }
            """;

    // ── POST /api/auth/logout ──────────────────────────────

    public static final String LOGOUT_SUCCESS_JSON = """
            {
              "code": "200-1",
              "message": "로그아웃 성공",
              "data": null
            }
            """;
}

File: src/main/kotlin/com/back/nbe9112team06/global/springDoc/example/AuthApiExamples.kt
package com.back.nbe9112team06.global.springDoc.example

object AuthApiExamples {
    // ── POST /api/auth/login ──────────────────────────────
    const val LOGIN_SUCCESS_JSON: String = """
            {
              "code": "201-1",
              "message": "로그인 성공",
              "data": {
                "nickname": "gildong"
              }
            }
            """

    // ── GET /api/auth/me ──────────────────────────────
    const val GET_MY_INFO_SUCCESS_JSON: String = """
            {
              "code": "200-1",
              "message": "조회 성공",
              "data": {
                "nickname": "gildong"
              }
            }
            """

    // ── POST /api/auth/logout ──────────────────────────────
    const val LOGOUT_SUCCESS_JSON: String = """
            {
              "code": "200-1",
              "message": "로그아웃 성공",
              "data": null
            }
            """
}